<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>525</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AB PLC Tag viewer         Author:HW   V1.0.1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form{background-image:url(&quot;./background.jpg&quot;)}</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AB PLC 变量数据监控软件</string>
   </property>
  </widget>
  <widget class="QLabel" name="PLC_IP_lable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PLC IP地址：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Dint_float_select">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>整型/浮点型</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="string_select">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>字符串</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IP_input">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>请输入PLC iP地址</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background-color:rgb(250, 240,210,75%);
				border-radius:5px}
QLineEdit:hover{background-color:rgb(200, 200,160,75%)}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Tag_input">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mangal</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>请输入PLC标签</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background-color:rgb(250, 240,210,75%);
border-radius:5px}
QLineEdit:hover{background-color:rgb(200, 200,120,75%)}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>158</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#logo{background-image:url(&quot;./PLC_logo.jpg&quot;);border-radius:15px}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="data_display">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>281</width>
     <height>421</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="179">
         <red>175</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="179">
         <red>175</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="179">
         <red>175</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="179">
         <red>175</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="179">
         <red>175</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="179">
         <red>175</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>监控到的PLC标签内容</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser#data_display{background-color:rgb(175, 240,210,70%);border-radius:8px}
QTextBrowser#data_display:hover{background-color:rgb(250, 200,120,75%)}</string>
   </property>
  </widget>
  <widget class="QLabel" name="line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>___________________________________________________________________________</string>
   </property>
  </widget>
  <widget class="QLabel" name="PLC_IP_lable_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据内容：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="monitor_start">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(150, 120,70,90%);border-radius:5px;border-style: outset}
QPushButton:hover{background-color:rgb(80, 120,70,90%)}
QPushButton:pressed{background-color:rgb(40, 100,70,90%)}</string>
   </property>
   <property name="text">
    <string>开始监控</string>
   </property>
  </widget>
  <widget class="QLabel" name="PLC_IP_lable_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Excel保存地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="excel_path">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="191">
         <red>250</red>
         <green>240</green>
         <blue>210</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Mangal</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>请输入想要保存的excel 文件地址及名称</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background-color:rgb(250, 240,210,75%);border-radius:5px}
QLineEdit:hover{background-color:rgb(90, 200,120,75%)}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shut_down">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(10, 100,200,85%);border-radius:5px;border-style: outset}
QPushButton:hover{background-color:rgb(80, 120,70,75%)}
QPushButton:pressed{background-color:rgb(60, 100,70,75%)}</string>
   </property>
   <property name="text">
    <string>关闭软件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_excel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(90, 210,100,85%);border-radius:5px;border-style: outset}
QPushButton:hover{background-color:rgb(80, 120,70,85%)}
QPushButton:pressed{background-color:rgb(60, 100,70,85%)}</string>
   </property>
   <property name="text">
    <string>导出Excel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="monitor_stop">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(250, 80,120,90%);border-radius:5px;border-style: outset}
QPushButton:hover{background-color:rgb(200,60,80,90%)}
QPushButton:pressed{background-color:rgb(200, 20,50,90%)}</string>
   </property>
   <property name="text">
    <string>停止监控</string>
   </property>
  </widget>
  <widget class="QLabel" name="PLC_IP_lable_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PLC变量</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>871</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
